charts/kube-vip-cloud-provider/templates/deployment.yaml

```diff
@@ -12,13 +12,20 @@ spec:
     metadata:
       labels:
         {{- include "kube-vip-cloud-provider.selectorLabels" . | nindent 8 }}
+      annotations:
+        configmap-checksum: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
     spec:
       containers:
         - command:
             - /kube-vip-cloud-provider
             - --leader-elect-resource-name=kube-vip-cloud-controller
           image: {{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
           name: {{ include "kube-vip-cloud-provider.name" . }}
+          env:
+          - name: KUBEVIP_NAMESPACE
+            value: {{ .Release.Namespace | default "kube-system" }}
```

Comment:
I believe releases always have a namespace, so the default "kube-system" can never be applied, so it could be removed (dead code).

https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
@rptaylor yes absolutely